lib/Terminal/Print/Grid.pm6

```diff
@@ -216,6 +218,8 @@ multi method print-cell($x, $y, %c) {
     self.print-cell($x, $y);
 }
 
+multi method print-cell($x, $y, *%c where *.elems) { samewith($x, $y, %c) }
```

Comment:
A where clause can be expensive (as can the samewith).

Reply:
Same as above. Without `where` it conflicts with `print-cell($x, $y)`. Because it's just for convenience, can be just removed.